components/brave_new_tab_ui/containers/newTab/stats.tsx

```diff
@@ -30,7 +26,7 @@ class Stats extends React.Component<Props, {}> {
   }
 
   get estimatedTimeSaved () {
-    const estimatedMillisecondsSaved = (this.adblockCount + this.trackedBlockersCount) * this.millisecondsPerItem || 0
+    const estimatedMillisecondsSaved = (this.adblockCount) * this.millisecondsPerItem || 0
```

Comment:
Total nitpick, but I think the parens are unnecessary now.

Reply:
👍 fixed.